src/home/homectl.c

```diff
@@ -3069,10 +3073,12 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case ARG_LUKS_VOLUME_KEY_SIZE:
                 case ARG_LUKS_PBKDF_PARALLEL_THREADS:
+                case ARG_LUKS_SECTOR_SIZE:
                 case ARG_RATE_LIMIT_BURST: {
                         const char *field =
                                        c == ARG_LUKS_VOLUME_KEY_SIZE ? "luksVolumeKeySize" :
                                 c == ARG_LUKS_PBKDF_PARALLEL_THREADS ? "luksPbkdfParallelThreads" :
+                                           c == ARG_LUKS_SECTOR_SIZE ? "luksSectorSize" :
```

Comment:
i'd prefer if we'd do some vaidation of this value ahead of time, i.e. here, just to print a friendly message. i.e. add a helper, valid_sector_size() that checks the value is `>= 512` and that `ISPOWEROF2()` holds. (the kernel internally usually also enforces the a check `<= page_size()`, but I would probably avoid that check, since page sizes in kernel and userspace might be quite different.